Arm/Insts/DPI/Add_sub_imm.lean

```diff
@@ -39,7 +39,7 @@ def exec_add_sub_imm (inst : Add_sub_imm_cls) (s : ArmState) : ArmState :=
 ----------------------------------------------------------------------
 
 /-- Generate random instructions of the DPI.Add_sub_imm class. -/
-def Add_sub_imm_cls.rand : IO (Option (BitVec 32)) := do
+def Add_sub_imm_cls.dpi.rand : IO (Option (BitVec 32)) := do
```

Comment:
DPI is one level above Add_sub_imm, how about calling it `Add_sub_imm.addsub.rand`?

Reply:
@nebeid @pennyannn I'd suggest `Add_sub_imm.inst.rand`, given that `Add_sub_imm_cls.rand` is just a list of this one function.